site/content/entry/yuzu-progress-report-jul-2021/index.md

```diff
@@ -0,0 +1,482 @@
+`Super Smash Bros. Ultimate`. A better solution was needed.
+
+toastUnlimited now offers us [the all new `Auto` CPU accuracy setting!](https://github.com/yuzu-emu/yuzu/pull/6573) 
+Enabled by default for all users, this setting determines the need to use the `Unfuse FMA` value automatically, by reading if the FMA instruction set is supported by the CPU 
+in 
+use.
+It also sets other values, for example `Faster ASIMD instructions`, to boost the performance of 32-bit games.
+Auto CPU accuracy has the potential to more than triple the performance of users running old or very low end CPUs!
+
+Thanks to work done by Morph, [now all default six Miis are available to the user](https://github.com/yuzu-emu/yuzu/pull/6556) in games that request them.
+
+{{< imgs
+	"./mii.png| We're still far away from offering full Mii customization, but at least more options are available now (Mario Kart 8 Deluxe)"
+  >}}
+
+[lioncash](https://github.com/yuzu-emu/yuzu/pull/6574), our silent optimizer and code cleaner, found an issue with the new strings that report if a game is 32 or 64-bit.
+Languages that read from right to left had issues with the initial implementation, and translating this string was disabled.
+[Now both issues are resolved!](https://github.com/yuzu-emu/yuzu/pull/6574)
+
+After gathering more information on the behaviour of the FPS unlimiter, epicboy discovered that some games will crash when attempting to boot them unlimited.
+[Simply force-enabling the limiter at each game boot is enough to solve the issue.](https://github.com/yuzu-emu/yuzu/pull/6576)
+Remember to unlock the framerate manually after you start a game!
+
+[vonchenplus](https://github.com/vonchenplus) is back with another nice addition!
+Certain game dumps contain several games inside them, and yuzu would default to only launching the first one in the list.
+[This change makes the necessary modifications so all games are properly listed.](https://github.com/yuzu-emu/yuzu/pull/6582)
+
+{{< imgs
+	"./games.png| "
+  >}}
+
+## Command-Line Shenanigans
+
+toastUnlimited did a general [update to the settings of the command-line version of yuzu](https://github.com/yuzu-emu/yuzu/pull/6651), `yuzu-cmd`.
+
+The previous implementation had many options that were originally carried over from Citra and later deprecated, as well as others that were not read properly from the `ini` 
+file, or were read but not written into the `ini` file, etc.
+In other words, there were a lot of things wrong with it, and some updating was in order to properly synchronize everything back with the settings present in the main 
+executable of yuzu.
+
+Since toastUnlimited was already working on `yuzu-cmd`, he also went on and [fixed some problems related to Vulkan on Linux](https://github.com/yuzu-emu/yuzu/pull/6652).
+When this executable was launched, it wouldn't be able to detect the window manager, and would proceed to exit instead of booting a game.
```

Comment:
```suggestion
When this executable was launched, it wasn't able to detect the window manager, and would proceed to exit instead of booting a game.
```